### PR TITLE
Fix golint failures of pkg/apis/autoscaling

### PR DESCRIPTION
**Pull request by SataQiu**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Fix golint failures of pkg/apis/autoscaling

**Which issue(s) this PR fixes**:

Ref #68026

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/apis/autoscaling/types.go [hunk 1]
```diff
@@ -226,8 +226,11 @@ type MetricTarget struct {
 type MetricTargetType string
 
 var (
-	UtilizationMetricType  MetricTargetType = "Utilization"
-	ValueMetricType        MetricTargetType = "Value"
+	// UtilizationMetricType is the type of utilization metric
+	UtilizationMetricType MetricTargetType = "Utilization"
+	// ValueMetricType is the type of value metric
+	ValueMetricType MetricTargetType = "Value"
+	// AverageValueMetricType is the type of average value metric
```
Comment (liggitt) [inline pkg/apis/autoscaling/types.go]: These aren't accurate… suggest checking the field documentation for where these are used for more accurate descriptions

Comment (SataQiu) [reply pkg/apis/autoscaling/types.go]: OK, I'll revise it later.

Comment (SataQiu) [reply pkg/apis/autoscaling/types.go]: I've updated the code, but it still doesn't feel quite right.
Can anyone give me some advice? @feiskyer @yifan-gu @luxas

**Review by thockin — COMMENTED** [review]
Thanks!

/lgtm
/approve

**Pull request merged**